tests/test_provenance.py

```diff
@@ -77,6 +80,14 @@ def decode(self, prov):
         ns = builder.build_classes()
         return ns.TskitProvenance.from_json(prov)
 
+    def verify_equal(self, ts1, ts2):
+        ts1_tables = ts1.dump_tables()
+        ts2_tables = ts2.dump_tables()
+        # We exclude provenance here due to timestamps
```

Comment:
I prefer 
```python
ts1_tables.provenances.clear()
ts2_tables.provenances.clear()
self.assertEqual(ts1_tables, ts2_tables)
```
since this means that we don't need to know which tables are present (and maybe we'll add some more to tskit some day).

ts.dump_tables() will always return a copy of the tables so it's safe to do (ts.tables might return a read-only view on the underlying tables someday)

Reply:
:+1: